pkg/apis/eksctl.io/v1alpha5/defaults.go

```diff
@@ -53,9 +53,9 @@ func SetClusterConfigDefaults(cfg *ClusterConfig) {
 	}
 }
 
-// IAMServiceAccountsWithAWSNodeServiceAccount returns the specified IAM service
+// IAMServiceAccountsWithImplicitServiceAccounts returns the specified IAM service
 // accounts including the potentially autocreated aws-node account as well
-func IAMServiceAccountsWithAWSNodeServiceAccount(cfg *ClusterConfig) []*ClusterIAMServiceAccount {
+func IAMServiceAccountsWithImplicitServiceAccounts(cfg *ClusterConfig) []*ClusterIAMServiceAccount {
```

Comment:
The name (and function) still confuses me a bit,  if I'm understanding the function correctly it appends to the cfg a service account for the AWS-Node, only if:
- the vpc cni isn't specified?
- the service account isn't found?

Reply:
yeah there are accounts that are implicitly in the list of service accounts and for aws-node if it isn't specified in the list (in the config) and the VPC CNI isn't handling it, then this function adds it

Reply:
ah I see that makes sense, could you update the comment to something like the above as it explains it well 😄